check_packages.py

```diff
@@ -1,99 +1,94 @@
 """
-Checking and reinstalling the external packages
+Check the existing environment against expected values
 """
 from __future__ import print_function
 
 import sys
+import yaml
```

Comment:
There should be a check whether the current environment has `yaml` installed. If not, prompt the user to install it, preferably showing the exact command to do so.